Makefile.Common

```diff
@@ -15,7 +15,7 @@ SRC_ROOT := $(shell git rev-parse --show-toplevel)
 # build tags required by any component should be defined as an independent variables and later added to GO_BUILD_TAGS below
 GO_BUILD_TAGS=""
 GOTEST_OPT?= -race -timeout 300s -parallel 4 --tags=$(GO_BUILD_TAGS)
-GOTEST_INTEGRATION_OPT?= -race -timeout 360s -parallel 4
+GOTEST_INTEGRATION_OPT?= -race -timeout 1200s -parallel 4
```

Comment:
reverted back to previous value 
previously I increased this because there was an error starting container in pipeline and I was testing it, forgot to return to previous value